nl/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -1,9 +1,9 @@
 <!ENTITY pbMacroSearchPanel.width "600">
 
 <!ENTITY insertMacro.key "l">
-<!ENTITY insertMacro.modifiers "accel">
+<!ENTITY insertMacro.modifiers "versnellen">
 
 <!ENTITY insertMacro.label "Quick Bar">
 
-<!ENTITY pbMacroSearchPanel.firstrunWin "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Options > Composition > Advanced">
-<!ENTITY pbMacroSearchPanel.firstrunMac "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Preferences > Composition > Advanced">
+<!ENTITY pbMacroSearchPanel.firstrunWin "Bij het intypen van een URL vult Postbox deze automatisch aan tegen de URLs uit uw e-mails. Om URL automatisch aanvullen uit te schakelen gaat u naar Opties > Compositie > Geavanceerd">
+<!ENTITY pbMacroSearchPanel.firstrunMac "Bij het intypen van een URL vult Postbox deze automatisch aan tegen de URLs uit uw e-mails. Om URL automatisch aanvullen uit te schakelen gaat u naar Opties > Compositie > Geavanceerd"">
```

Comment:
CORRECTIONS NEEDED. 
Was copy pasted from previous line. See Preferences vs Options.
Preferences could be translated with ''Voorkeuren'' or ''Instellingen'' depending on context.

Also, It's a literal, word-for-word translation. Although the words used are technically correct, it sounds a little off in Dutch. ''Tegen'' and ''Compositie'' wouldn't be used in this context.